scripts/common/docker.sh

```diff
@@ -178,6 +183,8 @@ docker_configure_proxy() {
         echo "Environment=\"HTTP_PROXY=${PROXY_ADDRESS}\" \"NO_PROXY=${NO_PROXY_ADDRESSES}\"" >> $file
     fi
 
+    log "Environment=\"HTTP_PROXY=${PROXY_ADDRESS}\" \"NO_PROXY=${NO_PROXY_ADDRESSES}\""
+
```

Comment:
```suggestion
```

Reply:
We have not this info in the logs today 
See that we just add to the file.
Would be not great we are able to check in the logs for the support cases?